js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,14 @@
+
+function aperture(n, list) {
+  const arrayContainer = [];
+  for (let i = 0; i < list.length; i++) {
```

Comment:
you can also put `list.length` in a separate variable and use that variable instead of `list.length`